packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.Props.ts

```diff
@@ -114,4 +116,126 @@ export interface ISpinButtonProps {
    * Icon for the decrement button of the spinButton
    */
   decrementButtonIcon?: IIconProps;
+
+  /**
+   * Custom styling for individual elements within the button DOM.
+   */
+  styles?: Partial<ISpinButtonStyles>;
+
+  /**
+   * Custom styles for the upArrow button.
+   *
+   * Note: The buttons are in a checked state when arrow keys are used to
+   * incremenent/decrement the spinButton. Use rootChecked instead of rootPressed
+   * for styling when that is the case.
+   */
+  upArrowButtonStyles?: Partial<IButtonStyles>;
+
+  /**
+   * Custom styles for the downArrow button.
+   *
+   * Note: The buttons are in a checked state when arrow keys are used to
+   * incremenent/decrement the spinButton. Use rootChecked instead of rootPressed
+   * for styling when that is the case.
+   */
+  downArrowButtonStyles?: Partial<IButtonStyles>;
+
+  /**
+   * Theme provided by HOC.
+   */
+  theme?: ITheme;
+}
+
+export interface ISpinButtonStyles {
+
+  /**
+   * Styles for the container of the spin button component.
+   */
+  container: IStyle;
```

Comment:
Per offline discussion, changed this to root. Changed what was previously root to be spinButtonWrapper.